src/main.rs

```diff
@@ -114,19 +118,62 @@ struct SyncOpts {
     ), value_delimiter = ',')]
     services: Vec<String>,
 
-    /// Path to a checkout of `rust-lang/team`.
-    #[clap(long, global(true), conflicts_with = "team_json")]
-    team_repo: Option<PathBuf>,
-
-    /// Path to a directory with prebuilt JSON data from the `team` repository.
-    #[clap(long, global(true))]
-    team_json: Option<PathBuf>,
-
+    /// Source of the data against which is the sync performed.
+    /// Possible values:
+    /// - in-tree => use the current team checkout
+    /// - production => use the live REST API team endpoint
+    /// - <path> => use prebuilt directory at <path>
+    #[arg(
```

Comment:
I wanted to implement it as two commands at first, but it was quite tricky in clap, especially since `DataSource::Prebuilt` actually takes a value (path).